my-app/src/context/UserContext.tsx

```diff
@@ -2,7 +2,7 @@ import { createContext } from "react"
 import { UserContentState } from "./UserContent"
 import { IUserContext } from "./IUserContext";
 
-export const UserContext = createContext<IUserContext>({ user: UserContentState, logIn: () => { }, logOut: () => { }, orderProducts: [], addProduct: () => { }, removeProduct: () => { }, increaseQuantity: () => { }, decreaseQuantity: () => { } });
+export const UserContext = createContext<IUserContext>({ user: UserContentState, logIn: () => { }, logOut: () => { }, orderProducts: [], addProduct: () => { }, removeProduct: () => { }, removeAllProducts: () => { }, increaseQuantity: () => { }, decreaseQuantity: () => { } });
```

Comment:
One context should have logic for one part, so that better option is to split user context and cart context